internal/expr/parser.go

```diff
@@ -203,6 +203,12 @@ loop:
 		}
 
 		p.pos++
+
+		// An opening parentheses might be the start of an expression.
+		if p.peekByte('(') {
+			break loop
+		}
```

Comment:
Its because its very common to do `INSERT INTO table (...) VALUES(...)` [e.g. in this sqlite tutorial](https://www.sqlitetutorial.net/sqlite-insert/). Advance looks for interesting chars that may *preceed* the expression but "(" is *part of* the expression so it needs to be handled separately.

It is also allowed to omit the space after the `table` so simply adding `"S"` to the list of interesting chars would not work.

Reply:
Makes sense, thanks